src/SFA.DAS.LevyTransferMatching.Web.UnitTests/Extensions/ApprenticeshipFundingDtoExtensionsTests.cs

```diff
@@ -94,11 +94,11 @@ public void CalcFundingForDate_Calculates_Correctly_For_2_Apprentices_24_Months(
             {
                 Duration = 24,
                 MaxEmployerLevyCap = 20_000,
-                EffectiveFrom = new DateTime(DateTime.UtcNow.Year, DateTime.UtcNow.AddMonths(-1).Month, 1),
+                EffectiveFrom = new DateTime(2021, DateTime.UtcNow.AddMonths(-1).Month, 1),
                 EffectiveTo = null
             };
 
-            var result = sut.CalcFundingForDate(2, new DateTime(DateTime.UtcNow.Year, 10, 1));
```

Comment:
Best to remove all DateTime.UtcNow usages and just use hardcoded values in all of this file.

Reply:
I have updated the  DateTime.UtcNow year to hardcoded year in unit tests.

Reply:
There are still some usages of DateTime.UtcNow like DateTime.UtcNow.AddMonths?

Reply:
@SHESFA Hi Sajid, Thanks for the comments. I totally agree with you but just like Chris's reviews, please can you give some suggestions on what should we do with DateTime.UtcNow.AddMonths in this case?

Ideally, this DateTime.UtcNow refactoring should have its own task separate from normal stories. I just made those few changes for the year 2021 so that I can compile my code and submit it for a PR.  Otherwise, the Main branch was broken with 2 tests where the year 2021 was required. What do you suggest? please let me know. At the moment, all other tests are passing. but you are correct we should change DateTime.UtcNow, at some point.

Reply:
@SHESFA As you suggested I have updated the tests. Removed the DateTime.UtcNow. All done in that one file.